### PR TITLE
Update CHANGELOG from m112 release

### PR DESCRIPTION
**Pull request by rachaprince**
Reference: http://cl/433250870

### CODE REVIEW

File: firebase-appdistribution/CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+* {{feature}} The Firebase App Distributio Android SDK is now available in beta. You
+  can use this SDK to notify testers in-app when a new test build is available.
+  To learn more, visit the
+  [reference documentation](/docs/reference/android/com/google/firebase/app-distribution/package-summary).
+
+#### Kotlin extensions
+The Android library with Kotlin extensions is now available in
+beta. The Kotlin extensions library transitively includes the base
+`firebase-app-distribution` library. To learn more, visit the
+[KTX reference documentation](/docs/reference/kotlin/com/google/firebase/app-distribution/ktx/package-summary).
```
Comment (lfkellogg) [inline firebase-appdistribution/CHANGELOG.md:17]: This link needs to point to docsite.

File: firebase-appdistribution/CHANGELOG.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+# Released
+
+## v20.0.6 (M112)
+
+#### Android library
+* {{feature}} The Firebase App Distributio Android SDK is now available in beta. You
+  can use this SDK to notify testers in-app when a new test build is available.
+  To learn more, visit the
+  [reference documentation](/docs/reference/android/com/google/firebase/app-distribution/package-summary).
```
Comment (lfkellogg) [inline firebase-appdistribution/CHANGELOG.md:11]: This link needs to point to docsite.

**Review by lfkellogg — APPROVED** [review]

File: firebase-appdistribution/CHANGELOG.md (hunk 1 above)
Comment (rachaprince) [reply firebase-appdistribution/CHANGELOG.md]: thanks!

File: firebase-appdistribution/CHANGELOG.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
 # Unreleased
-- Initial public beta release
\ No newline at end of file
+
+# Released
+
+## v20.0.6 (M112)
+
+#### Android library
+* {{feature}} The Firebase App Distributio Android SDK is now available in beta. You
```
Comment (mannyjimenez0810) [inline firebase-appdistribution/CHANGELOG.md:8]: nit: Distribution

**Review by mannyjimenez0810 — APPROVED** [review]

**Pull request merged**